src/contracts/pods/DelayedWithdrawalRouter.sol

```diff
@@ -107,14 +107,38 @@ contract DelayedWithdrawalRouter is Initializable, OwnableUpgradeable, Reentranc
         return _userWithdrawals[user];
     }
 
+    /// @notice Getter function to get all delayedWithdrawals of the `user`
+    function getUserDelayedWithdrawals(address user) external view returns (DelayedWithdrawal[] memory) {
+        uint256 delayedWithdrawalsCompleted = _userWithdrawals[user].delayedWithdrawalsCompleted;
+        uint256 totalDelayedWithdrawals = _userWithdrawals[user].delayedWithdrawals.length;
+        uint256 userDelayedWithdrawalsLength = totalDelayedWithdrawals - delayedWithdrawalsCompleted;
+        DelayedWithdrawal[] memory userDelayedWithdrawals = new DelayedWithdrawal[](userDelayedWithdrawalsLength);
+        for (uint256 i = 0; i < userDelayedWithdrawalsLength; i++) {
+            userDelayedWithdrawals[i] = _userWithdrawals[user].delayedWithdrawals[delayedWithdrawalsCompleted + i];
+        }
+        return userDelayedWithdrawals;
+    }
+
     /// @notice Getter function to get all delayedWithdrawals that are currently claimable by the `user`
-    function claimableUserDelayedWithdrawals(address user) external view returns (DelayedWithdrawal[] memory) {
+    function getClaimableUserDelayedWithdrawals(address user) external view returns (DelayedWithdrawal[] memory) {
```

Comment:
let's keep this function exactly the same to not break the UI. We can create a new function that does this. Prob would be good to just add a comment that this function is slightly misnamed.. :)